### PR TITLE
Reading list: set default reading time to 1 minute

### PR DESCRIPTION
**Pull request by akshayymahajan**
fixes 0 min reading time in reading list by setting the default reading time to 1 min

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Related Tickets & Documents

See #3217

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by akshayymahajan** [comment]
Fixes issue #3217

**Comment by rhymes** [comment]
Hi @akshayymahajan, thanks for this PR and for signing the CLA! I've updated the title just to make it clearer :)

File: app/javascript/src/components/ItemList/ItemListItem.jsx [hunk 1]
```diff
@@ -13,6 +13,9 @@ export const ItemListItem = ({ item, children }) => {
     tags: item.article_tags || item.reactable_tags,
   };
 
+  // update readingTime to 1 min if the reading time is less than 1 min
+  adaptedItem.readingTime = Math.max(1, adaptedItem.readingTime);
```
Comment (rhymes) [inline app/javascript/src/components/ItemList/ItemListItem.jsx]: Hi @akshayymahajan, `Math.max` is a great idea but there's a catch: if for any reason `readingTime` is `undefined` this will return `NaN`.

I think the best bet is: `Math.max(1, adaptedItem.readingTime || null)` because `Math.max(1, null)` returns 1 correctly.

It shouldn't happen in reality but it's better to be safe :D

**Comment by rhymes** [comment]
I'm adding this comment here. The tests are failing because they should have been failing a long time ago but...:

- at lines https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/src/components/ItemList/__tests__/ItemListItem.test.jsx#L15 and https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/src/components/ItemList/__tests__/ItemListItem.test.jsx#L32 the tests are passing the reading time as text strings, instead they should be `1 ` not "1 min read" because "min read" is already added by the component

- when "1 min read" encounters Math.max(1, "1 min read") it will return `NaN`

To recap:

- the components should be updated to declare `article_reading_time` and `reading_time` as numbers instead of strings
- the test should be updated accordingly 
- `Math.max()` should then work as you designed it

**Comment by akshayymahajan** [comment]
I assumed that it would always have a numeric value. Thanks @rhymes, I'll update the code and tests!
